src/TeleSign.Services/TeleSignService.cs

```diff
@@ -52,6 +52,7 @@ public class TeleSignService
                     IWebRequester webRequester,
                     string accountName = "default")
         {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
@bwishan I realize this might be an issue for code running on this same app domain but TeleSign has contacted us that we should explicitly communicate with their service over TLS1.2 thus this requirement would be an SDK requirement. I've been doing some research on how someone would go around this and set TLS1.2 just for this SDK but the only workaround I can find is loading the TeleSign DLL dynamically in its own app domain (but im still researching the downsides to this). @bwishan do you have any suggestions on getting around the requirement with no on the app domain?